GovUk.Frontend.Umbraco.ExampleApp/uSync/v9/Content/breadcrumbs.config

```diff
@@ -0,0 +1,139 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Content Key="49aa9979-672c-4b96-9a1d-a87e0947511d" Alias="Breadcrumbs" Level="2">
+  <Info>
+    <Parent Key="5e682cbe-b867-491a-955f-3382446d5663">Home</Parent>
+    <Path>/Home/Breadcrumbs</Path>
+    <Trashed>false</Trashed>
+    <ContentType>contentPage</ContentType>
+    <CreateDate>2024-12-02T15:07:18</CreateDate>
+    <NodeName Default="Breadcrumbs" />
+    <SortOrder>26</SortOrder>
+    <Published Default="true" />
+    <Schedule />
+    <Template Key="29bec9d4-d54e-4896-823a-f366eef5e5a6">ContentPage</Template>
+  </Info>
+  <Properties>
+    <blocks>
+      <Value><![CDATA[{
+  "layout": {
+    "Umbraco.BlockList": [
+      {
+        "contentUdi": "umb://element/e3ad23e23b6f438ea04a43419ecaf5af",
+        "settingsUdi": "umb://element/bad6212d3da243918e6c3514adfe28cc"
+      },
+      {
+        "contentUdi": "umb://element/a1ccf76c5a2b4d8d858870bbf6960504",
+        "settingsUdi": "umb://element/fd0bf21d38b04987a11163a2286e493d"
+      },
+      {
+        "contentUdi": "umb://element/b25cd44487c84d61a8686645eb7caafa"
+      },
+      {
+        "contentUdi": "umb://element/19cffef1cfab45d88328c0be64b12722",
+        "settingsUdi": "umb://element/25442e801b434dd59de81d26a3aefb46"
+      },
+      {
+        "contentUdi": "umb://element/12d27c8cf5664ed98acc72e60ae186b9"
+      },
+      {
+        "contentUdi": "umb://element/a6a5bbcb906c4d518fb304c738d98ff6",
+        "settingsUdi": "umb://element/3b223663cdcc4f7d9543f6cb083b304f"
+      }
+    ]
+  },
+  "contentData": [
+    {
+      "contentTypeKey": "e0876897-88f4-4dcf-ae0a-ec5a37e3a185",
+      "udi": "umb://element/e3ad23e23b6f438ea04a43419ecaf5af",
+      "text": {
+        "markup": "<h2 class=\"govuk-heading-l\">Breadcumbs&nbsp;</h2>\n<p>Assist users understand and move between the multiple levels of a website.</p>\n<p>The breadcrumbs components is based on <a href=\"https://design-system.service.gov.uk/components/breadcrumbs/\">Gov.Uk design guidelines</a></p>",
```

Comment:
When you create pages for components in the example app you need to:
- include a 'Caption' block, which you can copy from other pages
- include a 'Page heading' block, which you can copy from other pages. Copying it will automatically give you the line underneath the heading, which is missing on these pages. 
- you've added an h2 heading here which is the wrong level. You won't need this when you have 'Page heading'.
- 'Breadcrumbs' is misspelled in the title of each page.

If you're going to link back to the component page on the GOV.UK Design System website (we don't usually), then the link text should be the title of the page you're linking to.

Reply:
Thanks for the feedback! All actioned now